src/shared/schema/mix/actions/dash/delete-dashboard.ts

```diff
@@ -0,0 +1,23 @@
+import type {DeleteDashResult} from '../../..';
+import {getTypedApi} from '../../..';
+import {EntryScope} from '../../../..';
+import {createTypedAction} from '../../../gateway-utils';
+import {deleteDashArgsSchema, deleteDashResultSchema} from '../../schemas/dash/delete-dashboard';
+
+export const deleteDashboard = createTypedAction(
+    {
+        paramsSchema: deleteDashArgsSchema,
+        resultSchema: deleteDashResultSchema,
+    },
+    async (api, {lockToken, dashboardId}): Promise<DeleteDashResult> => {
+        const typedApi = getTypedApi(api);
+
+        await typedApi.us._deleteUSEntry({
+            entryId: dashboardId,
+            lockToken,
```

Comment:
lockToken is optional in scheme now. As far as I remember, if we don't have token we need to create it before pass it to us